definitions/ext-istio_service/golden_metrics.yml

```diff
@@ -0,0 +1,16 @@
+throughput:
+  title: Throughput
+  query:
+    select: rate(sum(istio_requests_total), 1 SECOND) WHERE reporter = 'destination'
+    from: Metric
+errorRate:
+  title: Error rate
+  query:
+    select: (filter(rate(sum(istio_requests_total), 1 minute), WHERE response_code LIKE '5%') / rate(sum(istio_requests_total), 1 minute)) * 100 WHERE reporter = 'destination'
```

Comment:
curious how the golden metric harvester would handle this query

Reply:
Because of the filter?  Looks like it's used elsewhere:  https://github.com/newrelic/entity-definitions/search?q=filter+filename%3Agolden_metrics.yml

Reply:
I retract my concern then 😄

Reply:
@rcrocker can you confirm that these are ok? (I know we may have a couple of problematic queries still in the definitions so just to be on the safe side)